www-client/qutebrowser/qutebrowser-1.11.1.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{3_6,3_7} )
+DISTUTILS_USE_SETUPTOOLS="rdepend"
+
+inherit desktop distutils-r1 eutils xdg-utils
+
+DESCRIPTION="A keyboard-driven, vim-like browser based on PyQt5 and QtWebEngine"
+HOMEPAGE="https://www.qutebrowser.org/ https://github.com/qutebrowser/qutebrowser"
+SRC_URI="https://github.com/${PN}/${PN}/releases/download/v${PV}/${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="scripts test"
+
+BDEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
```

Comment:
This is already handled by `DISTUTILS_USE_SETUPTOOLS` variable in this ebuild.

Reply:
True I will upgrade the branch